recipes/libwebm/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class LibwebmConan(ConanFile):
+    name = "libwebm"
+    description = "Library for muxing and demuxing WebM media container files"
+    topics = ("conan", "libwebm", "webm", "container", "demuxing", "muxing", "media", "audio", "video")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://chromium.googlesource.com/webm/libwebm/"
+    license = "BSD-3-Clause"
+
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_pes_ts": [True, False],
+        "with_new_parser_api": [True, False],
+    }
+
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_pes_ts": True,
+        "with_new_parser_api": False,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version])
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["ENABLE_WEBMTS"] = self.options.with_pes_ts
+        tc.variables["ENABLE_WEBM_PARSER"] = self.options.with_new_parser_api
+        tc.variables["ENABLE_WEBMINFO"] = False
+        tc.variables["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", default=True)
```

Comment:
```suggestion
```
The `CMAKE_POSITION_INDEPENDENT_CODE` is automatically filled by CMakeToolchain. https://docs.conan.io/2/reference/tools/cmake/cmaketoolchain.html?highlight=cmake_position_independent_code#generated-files

Sometimes, we really need to enforce it, because the upstream customizes that flag.